Formula/fifechan.rb

```diff
@@ -14,12 +15,15 @@ class Fifechan < Formula
 
   depends_on "cmake" => :build
   depends_on "allegro" => :recommended
-  depends_on "sdl" => :recommended
-  depends_on "sdl_image" => :recommended
+  depends_on "sdl2" => :recommended
```

Comment:
let's just get rid of all of the `=> :recomended` and make them required. no one has been using he without options anyway